mapping-for-navigation/_posts/0000-01-09-modeling-interesections-for-map-navigation.md

```diff
@@ -0,0 +1,191 @@
+
+Place the node of the intersection on the average center. Connect all incoming ways to this node. N°o need to create a triangle that would require adding a bunch of implicit turn restriction relation to specify mandatory and prohibited directions of movement.
+
+### 7. 3-way T-shape intersection (dual carriageway and bi-directional roads double point ⤫)
+
+| ✅ **Recommended**  | ❌ **Not recommended** |
+|---|---|
+| [![][56845a9e]][56845a9e] | [![][b739d579]][b739d579] |
+| (_[fullsize][56845a9e]_ \| _[wireframe][d74d5782] view_) | (_[fullsize][b739d579]_ \| _[wireframe][53488b50] view_)  |
+
+  [56845a9e]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point.png "Three way T-shape dual carriageway and bi-directional roads double point intersection (fullsize view)"
+  [b739d579]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_err.png "Incorrect drawing of the three way T-shape dual carriageway and bi-directional roads double point intersection (fullsize view)"
+  [d74d5782]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_wireframe.png "Three way T-shape dual carriageway and bi-directional roads double point intersection (wireframe view)"
+  [53488b50]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_wireframe_err.png "Incorrect drawing of the three way T-shape dual carriageway and bi-directional roads double point intersection (wireframe view)"
+
+-   This place ☝️ [on the map](https://www.openstreetmap.org/edit#map=19/37.74440/-122.45915) 🗺
+
+| ✅ **Recommended**  | ❌ **Not recommended** |
+|---|---|
+| [![][eb9d8681]][eb9d8681] | [![][3a4e88da]][3a4e88da] |
+| (_[fullsize][eb9d8681]_ \| _[wireframe][b5713acf] view_) | (_[fullsize][3a4e88da]_ \| _[wireframe][bc59adb4] view_)  |
+
+  [eb9d8681]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_2.png "Three way T-shape dual carriageway and bi-directional roads double point intersection (fullsize view)"
+  [3a4e88da]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_err_2.png "Incorrect drawing of the three way T-shape dual carriageway and bi-directional roads double point intersection (fullsize view)"
+  [b5713acf]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_wireframe_2.png "Three way T-shape dual carriageway and bi-directional roads double point intersection (wireframe view)"
+  [bc59adb4]: ../../images/intersection-modeling/3way_bidir_and_dual_carriageway_intersection_double_point_wireframe_err_2.png "Incorrect drawing of the three way T-shape dual carriageway and bi-directional roads double point intersection (wireframe view)"
+
+-   This place ☝️ [on the map](https://www.openstreetmap.org/edit#map=19/33.70318/-117.86810) 🗺
+
+Try to avoide creating sausage roads.
```

Comment:
A small segment of northbound South Main Street in this example is tagged as a two-way road in order for southbound South Main Street to turn left onto East Columbine Avenue. This is a subtle, clever approach, so we should call it out specifically.